docs/interacting-with-openshift.md

```diff
@@ -99,48 +99,58 @@ $ minishift logs
-node configuration of your OpenShift cluster.
+While OpenShift is running, you can view and change the master or the node configuration of your OpenShift cluster.
 
-You can view the current OpenShift master configuration (_master-config.yaml_) via:
+To view the current OpenShift master configuration (_master-config.yaml_), run the following command:
 
 ```shell
 $ minishift openshift config view
 ```
 
-For displaying the node configuration, you can specify the `target` flag.
-For more details about the `view` command refer to its [synopsis](./minishift_openshift_config_view.md).
+To show the node configuration instead of the master configuration, specify the `target` flag.
 
-Let's look at [Cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) (CORS) as an example for patching the OpenShift master configuration.
-Per default, OpenShift will only allow cross origin resource requests from the IP of the
-cluster as well as localhost. This is specified via the `corsAllowedOrigins` property in the
-[master configuration](https://docs.openshift.com/enterprise/3.0/admin_guide/master_node_configuration.html#master-configuration-files) (_master-config.yaml_). To change this value and allow
-cross origin requests from all domains, one can execute:
+For details about the `view` command, see the [synopsis](./minishift_openshift_config_view.md) command reference.
+
+### Use-case examples
+
+#### Configuring cross-origin resource sharing
+
+In this use-case example, you update the OpenShift master configuration to change the resource IP
+address in [cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) (CORS).
+
+By default, OpenShift only allows cross-origin resource requests from the IP address of the
+cluster or from localhost. You specify the IP address in the `corsAllowedOrigins` property of the
+[master configuration](https://docs.openshift.com/enterprise/3.0/admin_guide/master_node_configuration.html#master-configuration-files) (_master-config.yaml_).
+
+To change the property value and allow cross-origin requests from all domains,
+run the following command:
 
 ```
 $  minishift openshift config set --patch '{"corsAllowedOrigins": [".*"]}'
 ```
 
-Per default, the master configuration is targeted, but you can also patch the node configuration
-by specifying the `target` flag. For more details about the
-`set` command refer to its [synopsis](./minishift_openshift_config_set.md).
+#### Changing the OpenShift routing prefix
+
+In this use-case example, you change the OpenShift routing suffix in the master configuration.
+
+If you use a static routing suffix, you can set the `routing-suffix` flag as part of the
+[`start`](./minishift_start.md) command. However, by default Minishift uses a dynamic routing prefix
+based on [nip.io](http://nip.io/).
+
+When the IP address of the VM is apart of the routing suffix, for example
```

Comment:
> Please look at the original text and not just at the updated text, you will see that I'm trying to do an English grammar proofread on the text without changing the meaning

So where in the original text do I say that the IP address is _apart_ from the routing prefix? In fact I am saying quite the opposite - "In this case the VM's IP is part of the routing suffix". The IP _is part_ of the routing prefix. This is what makes it dynamic, since we are assigning IPs dynamically, so you don't know the exact routing prefix until provisioning is complete. Hence the routing prefix is dynamic or at least has an dynamic part, opposed to something like 'foo.bar.com'